### ISSUE TITLE
Bug of card_list API response uniqueness

### ISSUE THREAD

**Issue by ocknamo** [nachat1/MonacardHub#9]
## Summary
- card_list API response array is not unique list. 

## Example
"Monacute" cards reproduce this.
All monacute cards have same asset_longname "Monacute".
The API uses the asset_longname if it exists, so all Monacute response will be the same.

This may be a problem when used to display a list of cards. Or I think it may be a bug in itself.
And the response is different from the "card.mona.js" response.(https://card.mona.jp/api/card_list)

## Question

- Is this a bug? (or does not monacute follow the monacard 2.0 spec?)
- How is card.mona.jp implemented?

Thanks!

**Comment by nachat1** [comment]
Thank you for the question.
This is a bug.

I checked the Monaparty api. If asset is trueNFT,  'get_issuances' api response  asset_longname as assetgrop.
So we need to complement the information with 'get_assets_info'.

card.mona.jp use get_assets_info so it does not have bug.
MonacardHub use get_issuances so it has bug. And I fixed it.
https://github.com/nachat1/MonacardHub/commit/da6bb42edf8829d86ff4fbd445eb33e94d73d0f7
